test/specs/SCInteractionsSpec.e2e.ts

```diff
@@ -0,0 +1,29 @@
+// import SCInteractionsPage from "../pageobjects/SCIntereactionsPage";
```

Comment:
Ensure this is production-ready and add a TODO on top of this file using the text from line:29